CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Refinery Changelog
 
+## 1.14.1 2022-05-16
+
+### Fixes
+
+3558a28 Fix crash bug related to sharding (#455) | [@kentquirk](https://github.com/kentquirk)
```

Comment:
Dashes for a markdown list instead of commit IDs.

Reply:
Sigh. *someday* I'll get this right first time. Today is not that day.